### ISSUE TITLE
Manifest merger fails due to duplicate application theme attribute in AndroidManifest

### ISSUE THREAD

**Issue by jenzz** [appnexus/mobile-sdk-android#20]
Using the latest **v2.3.1** the following error occurs during compilation:

```
Error:(114, 9) Attribute application@theme value=(@style/Theme.MyAppTheme.Launch) from AndroidManifest.xml:114:9
	is also present at com.appnexus.opensdk:appnexus-sdk:2.3.1:19:78 value=(@style/AppTheme)
	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:9:5 to override
```

I've had a look at the `AndroidManifest.xml` file of the **appnexus-sdk** library. It applies the following theme:
`<application android:allowBackup="true" android:label="@string/app_name" android:theme="@style/AppTheme"/>`

That whole `<application>` tag should not exist. **appnexus-sdk** is a library. Not an application.

I can obviously workaround the problem by adding `tools:replace="theme"` to the `<application>` tag of our `AndroidManifest.xml` as suggested in the stack trace. However, it would be great to have this fixed at the source. So I suggest removing it entirely.

**Comment by anwzhang** [comment]
Thanks Jens, I will resolve this issue for next release.

**Comment by jenzz** [comment]
Thanks @anwzhang, but this is still an issue in the latest **v2.4**.

**Comment by josecu** [comment]
Hi Jens,
The fix was released in RC2.5 last week, can you confirm?

Thanks!
Jose

**Comment by jenzz** [comment]
Yep. Fixed in **v2.5**.
